resources/util.go

```diff
@@ -8,3 +8,15 @@ import (
 func GetCategory(r Resource) string {
 	return strings.Split(fmt.Sprintf("%T", r), ".")[1] // hackey
 }
+
+func EqualStringPtr(v1, v2 *string) bool {
+	if v1 == nil && v2 == nil {
+		return true
+	}
+
+	if v1 != nil || v2 != nil {
```

Comment:
Shouldn't this be: `v1 == nil || v2 == nil` ?

Iiuc it would now always return `false` if at least one of the two pointers is not equal to `nil`.

Reply:
true. fixed